src/pages/home/report/ReportActionCompose.js

```diff
@@ -44,7 +44,8 @@ class ReportActionCompose extends React.Component {
         this.comment = props.comment;
         this.state = {
             isFocused: false,
-            textInputShouldClear: false
+            textInputShouldClear: false,
+            isCommentEmpty: this.props.comment
```

Comment:
This should be a boolean and we can reference `props.comment` directly here.

e.g.

```js
isCommentEmpty: props.comment.length === 0
```